packages/bpk-tokens/tokens/ios.android.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Just the name? This file is just generated, it well never actually be used...I hope :D